cypress/support/commands.ts

```diff
@@ -183,6 +183,28 @@ Cypress.Commands.add('keyboardControls', (triggerKeys: any = {}, count = 1) => {
   }
 });
 
+/**
+ * Intercept all requests and return
+ * @param {array} intercepts - Array of intercepts to return
+ * return {array} - Array of intercepted request strings
+ * return {string} - Intercepted request string
+ */
+Cypress.Commands.add('interceptAny', (method = '/GET/POST/PUT/PATCH/', url = '/v1/*', wait = false, timeout = 10000) => {
```

Comment:
1. We need to add the types, wonder why you don't get error about the Array
2. Timeout is not used here, what's it for?